src/atlas/grid/CubedSphereGrid.h

```diff
@@ -0,0 +1,251 @@
+/*
+ * (C) Copyright 2020 UCAR
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#pragma once
+
+#include <string>
+
+#include "atlas/grid/Grid.h"
+#include "atlas/grid/detail/grid/CubedSphere.h"
+
+namespace atlas {
+
+//---------------------------------------------------------------------------------------------------------------------
+class CubedSphereGrid;
+class EquiDistCubedSphereGrid;
+class EquiAnglCubedSphereGrid;
```

Comment:
I'm not seeing where the classes `EquiDistCubedSphereGrid` and `EquiAnglCubedSphereGrid` are implemented. I don't think they have a place either as they don't expose any different API. They could only serve as a convenience constructor:
```
auto grid = EquiDistCubedSphereGrid( 32 );
```
but this could also be achieved already with
```
auto projection = Projection("cubedsphere_equidistant", params);
auto grid = CubedSphereGrid( 32, projection ); // previous line could be inlined here
```

Reply:
Removed the other classes